test-services/minikube/start-minikube.sh

```diff
@@ -45,9 +45,6 @@ MINIKUBE_OPTIONS="--vm-driver=none --bootstrapper=${BOOTSTRAPPER} --kubernetes-v
 KUBEADM_OPTIONS="--feature-gates=CoreDNS=true \
     --extra-config=kubelet.authorization-mode=AlwaysAllow \
     --extra-config=kubelet.anonymous-auth=true"
-if [[ "$K8S_VERSION" =~ ^v1\.11\. ]]; then
```

Comment:
@jchengsfx do you remember why this was there?  We shouldn't really be using the cadvisor port for stats anymore anyway.

Reply:
In minikube, cadvisor was enabled by default for k8s versions <1.11, disabled by default in 1.11, and then removed in 1.12.  We have a cadvisor test for it (now skipped if testing in k8s >=1.12). If this is now unnecessary, we can delete this and disable/remove the test.